File: 성민/레벨2/카펫.js
function solution(brown, yellow) {
  var answer = [];
  for (let i = 1; i <= yellow; i++) {
    if ((2 * yellow) / i + 4 + 2 * i === brown) {
      const a = i + 2;
      const b = (yellow + brown) / (i + 2);
      return a > b ? [a, b] : [b, a];
    }
  }
  return answer;
}

console.log(solution(10, 2));

File: 종우/카펫.py
# 1008_4
# 카펫
# https://school.programmers.co.kr/learn/courses/30/lessons/42842

# 내 풀이, 2차 방정식으로 품
def solution(brown, yellow):
    x = (4+brown+((4+brown)**2-16*(yellow+brown))**0.5)/4
    y = (4+brown-((4+brown)**2-16*(yellow+brown))**0.5)/4
    return [x,y]

